plaid/accounts.go

```diff
@@ -80,15 +82,16 @@ type getAccountsRequest struct {
 type GetAccountsResponse struct {
 	APIResponse
 	Accounts []Account `json:"accounts"`
-	Item Item `json:"item"`
+	Item     Item      `json:"item"`
```

Comment:
nit: extra specing here

Reply:
This spacing is a gofmt change (aligns all fields).  I can revert, but it'd actually be non-standard.